base/range.jl

```diff
@@ -860,13 +860,19 @@ end
 
 median(r::Range{<:Real}) = mean(r)
 
-function in(x, r::Range)
+function in(x::Real, r::Range{<:Real})
+    n = step(r) == 0 ? 1 : round(Integer,(x-first(r))/step(r))+1
```

Comment:
Given that the definitions are so close, I doubt it. Also it turns out it's harder than it seems, since their signatures are different: `T` doesn't need to be the same for both arguments with the `Real` method.